couchbase/admin/search/v1/search.proto

```diff
@@ -12,11 +12,35 @@ service SearchAdminService {
   rpc GetIndex(GetIndexRequest) returns (GetIndexResponse) {}
   rpc ListIndexes(ListIndexesRequest) returns (ListIndexesResponse) {}
 
-  rpc UpsertIndex(UpsertIndexRequest) returns (UpsertIndexResponse) {}
+  rpc CreateIndex(CreateIndexRequest) returns (CreateIndexResponse) {}
+  rpc UpdateIndex(UpdateIndexRequest) returns (UpdateIndexResponse) {}
```

Comment:
So the SDK would choose which to use based on whether the SearchIndex from the user has a UUID or not, I assume?

Reply:
Yes that was my thinking. I'm not tied to the idea but I think that the interface is nicer this way.

Reply:
Yep, makes sense